templates/default.json

```diff
@@ -5,13 +5,16 @@
     "@babel/preset-env",
     "@commitlint/cli",
     "@commitlint/config-angular",
+    "@testing-library/jest-dom",
+    "@testing-library/react",
     "@semantic-release/changelog",
     "@semantic-release/commit-analyzer",
     "@semantic-release/git",
     "@semantic-release/github",
     "@semantic-release/npm",
     "babel-eslint",
     "babel-jest",
+    "babel-plugin-jsx-remove-data-test-id",
```

Comment:
necessary to remove data-testid props when compiling for production

Reply:
@KevinDanikowski did you generate a project and then run `npm test` within that generated project to make sure the `babel-plugin-jsx-remove-data-test-id` plugin does what it says it does? Specifically I'm looking at this note from their readme:
> In some configurations the above will strip out the test attribute before the tests are run, causing them to fail. If this is the case for your project, you'll need to limit the plugin to non-test environments.

Reply:
Confirmed that it's gone and no issues with using them when testing